### ISSUE TITLE
Remove width:100% from component stylesheet

### ISSUE THREAD

**Issue by rolfsmeds** [gatanaso/multiselect-combo-box#45]
The component should probably not set itself to be 100% by default.

**Comment by gatanaso** [comment]
Thanks for the suggestion @rolfsmeds. I did a quick local test for this suggestion by removing the 100% width and the component behaves as follows (notice the variable width):
![mscb-01](https://user-images.githubusercontent.com/15094658/67889835-df5e5400-fb4f-11e9-8145-6c7df98ad121.gif)

What do you think about this? :)

**Comment by rolfsmeds** [comment]
Looks good to me. I think it's fine that it varies in width if you haven't defined a width for it. The benefit is that it doesn't take more space than it actually needs. And if you want the width to stay fixed, you set a width on it.

**Comment by rolfsmeds** [comment]
I guess it wouldn't hurt to expose a variable for the width of the input area. That way, if I know the placeholder text will be as short as "Add", I could easily set the input area to be e.g. 4em or so, so that it only takes as much space as it needs.

**Comment by gatanaso** [comment]
Thanks for taking the time to answer. Yes, I think introducing this change would be good and I can also look into exposing the variable for the input field as well.

**Comment by gatanaso** [comment]
Hi @rolfsmeds 

If left unspecified, the width of the input field container will default to what is specified by the `vaadin-text-field` which is:
```
width: var(--vaadin-text-field-default-width, 12em);
```
therefore to specify the text field container width, one can later define it as:
```
multiselect-combo-box {
  --vaadin-text-field-default-width: 500px;
}
```
I believe this should be flexible enough for most use cases.

**Comment by rolfsmeds** [comment]
Ah, of course, yes, that should do the trick!

**Comment by gatanaso** [comment]
fixed with #46
